app/api/api_v1/routers/search.py

```diff
@@ -188,7 +188,8 @@ def download_all_search_documents(db=Depends(get_db)) -> RedirectResponse:
             detail="Missing required environment variables",
         )
 
-    data_dump_s3_key = "navigator/whole_data_dump.csv"
+    s3_prefix = "navigator"
+    data_dump_s3_key = f"{s3_prefix}/whole_data_dump-{INGEST_CYCLE_START}.zip"
```

Comment:
Could we have these in their own subfolder for bucket-tidiness reasons? Perhaps like this: 
```suggestion
    data_dump_s3_key = f"{s3_prefix}/dumps/{INGEST_CYCLE_START}.zip"
```